src/domain/space/layout/tabbedLayout/Tabs/SpaceDashboard/SpaceDashboardPage.tsx

```diff
@@ -73,15 +76,14 @@ const SpaceDashboardPage = () => {
 
   return (
     <>
-      <PageContent>
-        {!loading && (
-          <ApplicationButtonContainer spaceId={spaceId}>
-            {(applicationButtonProps, loading) => {
-              if (loading || applicationButtonProps.isMember) {
-                return null;
-              }
-
-              return (
+      {!loading && (
+        <ApplicationButtonContainer spaceId={spaceId}>
+          {(applicationButtonProps, loading) => {
+            if (loading || applicationButtonProps.isMember) {
+              return null;
+            }
+            return (
+              <PageContent gridContainerProps={{ paddingBottom: 0 }} sx={{ flexGrow: 0 }}>
```

Comment:
This is coming from: https://github.com/alkem-io/client-web/pull/8207 but I applied it as I am touching the same file.